hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/HazelcastSqlToRelConverter.java

```diff
@@ -174,6 +176,17 @@ private RexNode convertCast(SqlCall call, Blackboard blackboard) {
         return getRexBuilder().makeCast(to, convertedOperand);
     }
 
+    /**
+     * This method overrides Apache Calcite approach to convert
+     * IN operator to chain of disjunctive comparison predicates.
+     * @see org.apache.calcite.sql2rel.SqlToRelConverter#substituteSubQuery
+     * @see org.apache.calcite.sql2rel.SqlToRelConverter#convertInToOr
+     *
+     */
+    private RexNode convertIn(SqlCall call, Blackboard blackboard) {
```

Comment:
I prepared this method for case if Hazelcast operators should not override Calcite's operators.